gpflow/features.py

```diff
@@ -0,0 +1,184 @@
+# Copyright 2017 st--, Mark van der Wilk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.from __future__ import print_function
+
+from abc import ABCMeta, abstractmethod
+from functools import singledispatch
+
+import numpy as np
+import tensorflow as tf
+
+from . import conditionals, transforms, kernels, decors, settings
+from .params import Parameter, Parameterized
+
+
+class InducingFeature(Parameterized, metaclass=ABCMeta):
```

Comment:
Do not use `ABCMeta` class here, Parameterized already has it with extended version.